File: Ineval.DAL/Entities/configuration/EmailParametros.cs

using System.ComponentModel.DataAnnotations.Schema;

namespace Ineval.DAL
{
    [Table("EmailParametros")]
    public class EmailParametros : GeneralConfigurationBase
    {
        public string EmailPrincipal { get; set; }
        public string EmailPassword { get; set; }
        public string EmailCopia { get; set; }

    }
}

File: Ineval/Scripts/Pages/core/configuration/EmailParametros.js
function emailparametrosCallback(data) {
    $('#Id').val(data.Id);
    $('#EmailPrincipal').val(data.EmailPrincipal);
    $('#EmailPassword').val(data.EmailPassword);
    $('#EmailCopia').val(data.EmailCopia);
    setSelectEmail("#EmailCopia", data.EmailCopia);
}
crearSelectizeEmail(['EmailCopia'], 4);